libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -80,17 +87,17 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
     .wrapper {
       position: relative;
       display: inline-flex;
-      align-items: center;
+      align-items: flex-start;
       vertical-align: top;
       padding-right: $spacing-to-label; // Ensure same padding on right side of label
-      min-height: map-get($checkbox-radio-sizes, 'md');
+      @include wrapper-top-bottom-padding(map-get($checkbox-radio-sizes, 'md'));
```

Comment:
Consider extracting `map-get($checkbox-radio-sizes, 'md')` to named variable:
`$default-checkbox-radio-size: map-get($checkbox-radio-sizes, 'md');`

Reply:
Done 😀 e9f9de5